pygmt/tests/test_clib_virtualfiles.py

```diff
@@ -80,7 +80,7 @@ def test_virtual_file_fails():
     with clib.Session() as lib, mock(lib, "GMT_Open_VirtualFile", returns=1):
         with pytest.raises(GMTCLibError):
             with lib.open_virtual_file(*vfargs):
-                print("Should not get to this code")
+                pass
```

Comment:
Should we do like a `raise ValueError("Should not get to this code")` instead of silently passing? Not that the code should ever get to it, but just in case?

Reply:
It's unnecessary. The test will fill if `lib.open_virtual_file` doesn't raise the `GMTCLibError` error.

Here is a minimal test to verify it:
```
import pytest

def test_zero_division():
    with pytest.raises(ZeroDivisionError):
        1 / 0

def test_zero_division_pass():
    with pytest.raises(ZeroDivisionError):
        pass
```
The output is:
```
========================================================================================== FAILURES ===========================================================================================
___________________________________________________________________________________ test_zero_division_pass ___________________________________________________________________________________

    def test_zero_division_pass():
>       with pytest.raises(ZeroDivisionError):
E       Failed: DID NOT RAISE <class 'ZeroDivisionError'>

test.py:8: Failed
=================================================================================== short test summary info ===================================================================================
FAILED test.py::test_zero_division_pass - Failed: DID NOT RAISE <class 'ZeroDivisionError'>
```

Reply:
Ok then!